terraform/main.tf

```diff
@@ -120,31 +155,12 @@ module "s3" {
   regional_account_id = var.regional_account_id
 }
 
-resource "aws_s3_bucket" "droits-wreck-images" {
-  bucket = "droits-wreck-images"
-  # Stops terraform from destroying the object if it exists
-  lifecycle {
-    prevent_destroy = true
-  }
-}
-
-resource "aws_s3_bucket_acl" "droits-wreck-images-acl" {
-  bucket = "droits-wreck-images"
-  acl    = "private"
+resource "aws_ecr_repository" "droits-webapp-repository" {
+  name         = var.webapp_ecr_repository_name
+  force_delete = true
 }
 
-resource "aws_s3_bucket_versioning" "droits-wreck-images-versioning" {
-  bucket = "droits-wreck-images"
-  versioning_configuration {
-    status = "Enabled"
-  }
-}
-
-resource "aws_s3_bucket_server_side_encryption_configuration" "droits-wreck-images-encryption-config" {
-  bucket = "droits-wreck-images"
-  rule {
-    apply_server_side_encryption_by_default {
-      sse_algorithm = "AES256"
-    }
-  }
-}
+resource "aws_ecr_repository" "droits-api-backoffice-repository" {
```

Comment:
We don't want the repos to be managed by TF, we just want to reference manually added ones. The reason for this being it will fail on first startup because we don't have a container to push anything into, so we reach a catch 22 issue. 

The fix to this would be having a pre-script, but it's not worth the effort when it's only done and needed once.